cpp/server/ct-server.cc

```diff
@@ -202,11 +202,15 @@ void SequenceEntries(TreeSigner<LoggedCertificate>* tree_signer,
 
   while (true) {
     if (election->IsMaster()) {
-      store->CleanupOldEntries();
-
-      // TODO(alcutter): Probably don't need to blow up here
-      CHECK_EQ(util::Status::OK, tree_signer->SequenceNewEntries())
-          << "Problem sequencing new entries";
+      util::Status status(store->CleanupOldEntries());
+      if (!status.ok()) {
+        LOG(WARNING) << "Problem cleaning up old entries: " << status;
+      } else {
+        status = tree_signer->SequenceNewEntries();
```

Comment:
I was hoping for a bribe of some kind at the very least.
Fair point about hitting etcd vs getting stuck, I'll change it.